Source/WebKit/UIProcess/WebScreenOrientationManagerProxy.cpp

```diff
@@ -150,13 +150,13 @@ void WebScreenOrientationManagerProxy::unlock()
 
     bool didUnlockOrientation = false;
 #if ENABLE(FULLSCREEN_API)
-    if (m_page.fullScreenManager() && m_page.fullScreenManager()->isFullScreen()) {
-        m_page.fullScreenManager()->unlockFullscreenOrientation();
+    if (m_page->fullScreenManager() && m_page->fullScreenManager()->isFullScreen()) {
```

Comment:
`fullScreenManager()` is not a trivial function so we need to protected the page.

Reply:
sure, fixed.